src/components/room/RoomActiveRequestEditor.vue

```diff
@@ -95,12 +95,17 @@ const rejectRequest = (reservationRequestId: number) => {
                 ${{ slotProps.data.cost }}
             </template>
         </Column>
-        <Column style="width: 4rem">
+        <Column field="guestCancelCount">
+            <template #header>
+                <i class="pi pi-ban"></i> Guest cancellations
+            </template>
+        </Column>
+        <Column style="width: 6rem">
```

Comment:
You use the `guestCancelCount` field here but it's not defined inside `ReservationRequestDTO` in `src/api/reservation.api.ts` (I guess it worked either way because it's Typescript, even though that seems like an oversight of its type system...).